frontend/src/components/SignForms/SignInForm/SignIn.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+
+import EmailField from './SignComponents/EmailField';
+import PasswordField from './SignComponents/PasswordField';
+import SignLabel from './SignComponents/SignLabel';
+import { SignInFormik } from '../SignInterfaces';
+
+import { SignWrapper, FormBox, ItemsBox, GridWrapper, GridSignInput, GridButton } from './styled';
+
+const SignIn = ({ formik }: SignInFormik) => {
+  const {
+    values: { login, password },
+    errors,
+    touched,
+    handleChange,
+    isValid,
+    setFieldTouched,
+    handleBlur,
+    handleSubmit,
+  } = formik;
+
+  const onChangeAccum = (name: boolean, e: string): any => {
```

Comment:
any?
What is Accum?
What is true and false sequences? Do we really need this method?

Reply:
renamed onChangeAccum => onChangeHandler.

Reply:
@k-krasilnikova renamed "onChangeAccum => onChangeHandler", any replaced. On my opinion its elegant approach for work with input data and errors if input data wrong. Lets see, Usually when the User starting enter the data called onChange and after we will handle result and show in additional filed under input error info or other info. "_<input ..../> + isError <p>Error</p>_". But this meta work otherwise - we will bind onChangeHandler and after every User tap will call onChangeHandler after/im call onChange and finally calling setFieldTouched. This method has 3 arg - name, true, false. **Name** - its input name for identificate. **True** - its isTouched status(if user starting enter the data isTouched anyway will be true) and last arg - **False** its helper which explicitly prevent/skip validation for every tap. Magic in "helperText" prop - helperText={touched.fieldname && errors.fieldname). each keyboard click in input will call onChangeHandler and based on input data and will observe - if smth wrong in touched(touched will be is not empty)  immediately helperText show to User wrong if all right - touched will be empty. We dont need to create additional fields for work as like paragraph and. also there are not problem with fields and code will be clear - this method fill only 3 lines. fuuh)) Lets save it?)